CHANGELOG.md

```diff
@@ -1,55 +1,84 @@
 # Changelog
 
+## ## [1.189.0] - 2019-07-16
```

Comment:
Please don't edit the CHANGELOG

Reply:
Regarding the README, I can see the changes so please remove them.

Reply:
done